en/docs/adv_examples/resnet.md

```diff
@@ -1,373 +1,353 @@
---output_directory
-# 指定转换后的ofrecord存储位置
- --num_threads
-# 任务运行线程数
---shards
-# 指定ofrecord分片数量，建议shards = 256
-#（shards数量越大，则转换后的每个ofrecord分片数据量就越少）
---bounding_box_file
-# 该参数指定的csv文件中标记了所有目标box的坐标，使转换后的ofrecord同时支持分类和目标检测任务
-```
-
-运行以上脚本后，你可以在../data/imagenet/ofrecord/validation、../data/imagenet/ofrecord/train下看到转换好的ofrecord文件：
-
-```shell
-. ├── train
-│   ├── part-00000
-│   └── part-00001
-                             ... └── validation
-    ├── part-00000
-    └── part-00001
-                             ...
-```
 
+#### How to generate ONNX model
 
+We have introduced how to convert OneFlow's ResNet model to ONNX model and give an example of using the onnx runtime to make predictions in above example. Similarly, you can follow the steps to complete the transformation of your training ResNet model or other models. 
 
-如果您尚未下载过Imagenet数据集，请自行下载和准备以下文件：
+**Step1: Save the model's weight**
 
-- ILSVRC2012_img_train.tar
+First you should specify the OneFlow model path, and then specify the transformed ONNX model storage path, like the following example. 
 
-- ILSVRC2012_img_val.tar
+首先指定待转换的OneFlow模型路径，然后指定转换后的ONNX模型存放路径，例如示例中：
 
-我们将用以下两个步骤，帮您完成数据集的预处理。我们将用以下两个步骤，帮您完成数据集的预处理。之后，您就可以使用上面介绍的转换脚本进行OFReciord的转换了。下面假设您已经下载好了原始数据集，并存放在data/imagenet目录下：下面假设您已经下载好了原始数据集，并存放在data/imagenet目录下：
-
-```shell
-├── data
-│   └── imagenet
-│       ├── ILSVRC2012_img_train.tar
-│       ├── ILSVRC2012_img_val.tar
-├── imagenet_utils
-│   ├── extract_trainval.sh
-│   ├── imagenet_2012_bounding_boxes.csv
-│   ├── imagenet_2012_validation_synset_labels.txt
-│   ├── imagenet_lsvrc_2015_synsets.txt
-│   ├── imagenet_metadata.txt
-│   ├── imagenet_ofrecord.py
-│   └── preprocess_imagenet_validation_data.py
-```
-
-**步骤一：extract imagenet**
-
-解压后，文件夹结构示意如下： . ├── extract_trainval.sh ├── imagenet │   ├── ILSVRC2012_img_train.tar │   ├── ILSVRC2012_img_val.tar │   ├── train │   │   ├── n01440764 │   │   │   ├── n01440764_10026.JPEG │   │   │   ├── n01440764_10027.JPEG ... │   │   └── n01443537 │   │       ├── n01443537_10007.JPEG │   │       ├── n01443537_10014.JPEG ... │   └── validation │       ├── ILSVRC2012_val_00000236.JPEG │       ├── ILSVRC2012_val_00000262.JPEG ...train文件夹下是1000个虚拟lebel分类文件夹(如：n01443537)，训练集图片解压后根据分类放入这些label文件夹中；validation文件夹下是解压后的原图。
-
-```shell
-sh extract_trainval.sh ../data/imagenet # 参数指定存放imagenet元素数据的文件夹路径
+```python
+#set up your model path
+flow_weights_path = 'resnet_v15_of_best_model_val_top1_77318'
+onnx_model_dir = 'onnx/model'
 ```
 
-```shell
-这一步主要是将ILSVRC2012_img_train.tar和ILSVRC2012_img_val.tar解压缩，生成train、validation文件夹。train文件夹下是1000个虚拟lebel分类文件夹(如：n01443537)，训练集图片解压后根据分类放入这些label文件夹中；validation文件夹下是解压后的原图。
-```
+**Step2: Create a new job function for inference**
 
-**步骤二：validation数据处理**
+Then, we create a new job function for inference, which only contains the network structure, except the operator to read the OFRecord, and accepts the form of numpy array input. You can refer to the `InferenceNet` in `resnet_to_onnx.py`. 
```

Comment:
except the operator to read the OFRecord
->
does not contain the operator to read the OFRecord